pulp_container/app/registry_api.py

```diff
@@ -309,13 +312,21 @@ def get_pull_through_drv(self, path):
         if not pull_through_cache_distribution:
             raise RepositoryNotFound(name=path)
 
+        pull_through_remote = models.ContainerPullThroughRemote.objects.filter(
+            pk=pull_through_cache_distribution.remote_id
+        ).values()[0]
+        if not filter_resource(
+            path, pull_through_remote.get("includes"), pull_through_remote.get("excludes")
```

Comment:
We may want to run the validation against `upstream_name` (i.e., repository name), not `path`. `path` includes the namespace of the caching distribution that is not transparent to the remote registry.